src/main/java/org/dependencytrack/persistence/DefaultObjectGenerator.java

```diff
@@ -48,6 +48,24 @@ public class DefaultObjectGenerator implements ServletContextListener {
 
     private static final Logger LOGGER = Logger.getLogger(DefaultObjectGenerator.class);
 
+    static final String DEFAULT_ADMIN_USERNAME = "admin";
+
+    static final String ADMIN_USERNAME_ENV_VARIABLE = "DEPENDENCY_TRACK_ADMIN_USERNAME";
```

Comment:
Should it be made even more clear that this is just about the initial value for these variables, i.e. `DEPENDENCY_TRACK_INITIAL_ADMIN_USERNAME`? To prevent people on Slack asking why the "change of the admin password env variable doesn't work".

Reply:
Yes good idea !